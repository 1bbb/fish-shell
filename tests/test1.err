functions: Function 'bar' already exists. Cannot create copy 'foo'
Standard input: functions -c foo bar
                ^

       functions - print or erase functions

   [1mS[30m[m(B[1my[30m[m(B[1mn[30m[m(B[1mo[30m[m(B[1mp[30m[m(B[1ms[30m[m(B[1mi[30m[m(B[1ms[30m[m(B
       functions [-e] FUNCTIONS...

functions: Type ‚Äúhelp functions‚Äù for related documentation

