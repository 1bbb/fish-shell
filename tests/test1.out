1a
1b
2a
2b
Test 2 pass
Test 3 pass
Test 4 pass
Test 5 pass
Test 6 pass
